### ISSUE TITLE
RDF dungeon cooldown

### ISSUE THREAD

**Issue by H4Xx0R-PC** [chromiecraft/chromiecraft#382]
#### WHAT CLIENT DO YOU PLAY ON?

- [ ] enGB
- [x] enUS
- [ ] other (specify)

##### FACTION

- [x] Alliance
- [ ] Horde

##### CONTENT PHASE:

- [x] Generic
- [ ] 1-19
- [ ] 20-29
- [ ] 30-39

##### CURRENT BEHAVIOUR:

When you enter a dungeon through RDF you get a 15 minute cooldown before you are able to queue for RDF again (https://www.wowhead.com/spell=71328/dungeon-cooldown). This 15 minute timer is supposed to clear when you finish the last boss of the dungeon you're running.

##### EXPECTED BLIZZLIKE BEHAVIOUR:

The 15min cooldown from RDF is supposed to reset after you clear the dungeon. For evidence check WoWhead comments on https://www.wowhead.com/spell=71328/dungeon-cooldown

##### STEPS TO REPRODUCE THE PROBLEM:

1. Queue for RDF
2. Clear dungeon in 5 minutes
3. Wait for 10 minutes before you can queue again

##### EXTRA NOTES:

https://www.wowhead.com/spell=71328/dungeon-cooldown

##### AC HASH/COMMIT:

https://github.com/chromiecraft/azerothcore-wotlk/commit/ee1918a3e9bc355fbd6f8d58d77760fb61fdbff7

##### OPERATING SYSTEM:

Ubuntu 20.04

##### MODULES:

- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-LevelUpReward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-CarbonCopy](https://github.com/55Honey/Acore_CarbonCopy)

##### OTHER CUSTOMIZATIONS:

None.

##### SERVER:

ChromieCraft

**Comment by mdawx** [comment]
Duplicate of #294
commit chromiecraft/azerothcore-wotlk@ee1918a does not include my fix:

```
git log --oneline | egrep "(ee1918a|462bbc6)"
462bbc6b fix(Core): Remove RDF cooldown after completion (#5034)
ee1918a3 fix(Core/Log): GMLogFile -> GmLogFile (#5130)
```

**Comment by Helias** [comment]
should be live on next update so

**Comment by H4Xx0R-PC** [comment]
> 
> 
> Duplicate of #294
> commit [chromiecraft/azerothcore-wotlk@ee1918a](https://github.com/chromiecraft/azerothcore-wotlk/commit/ee1918a) does not include my fix:
> 
> ```
> git log --oneline | egrep "(ee1918a|462bbc6)"
> 462bbc6b fix(Core): Remove RDF cooldown after completion (#5034)
> ee1918a3 fix(Core/Log): GMLogFile -> GmLogFile (#5130)
> ```

Ah, I used the search term "RDF" to search for duplicates which is why I couldn't find #294 .
